nemo_rl/algorithms/dpo.py

```diff
@@ -265,6 +273,49 @@ def add_ref_logprobs_to_data(dataloader, policy, master_config, is_val=False):
 # Training & Validation
 # =======================================================
 def validate(
+    policy: PolicyInterface,
+    val_dataloader: dict[str, StatefulDataLoader],
+    tokenizer,
+    loss_fn,
+    step: int,
+    master_config: MasterConfig,
+    val_batches: int,
+    val_batch_size: int,
+    val_mbs: int,
+    logger: Logger,
+):
+    val_metrics, validation_timings = {}, {}
+    for k, v in val_dataloader.items():
+        k_val_metrics, k_validation_timings = validate_one_dataset(
+            policy,
+            v,
+            tokenizer,
+            loss_fn,
+            step,
+            master_config,
+            val_batches,
+            val_batch_size,
+            val_mbs,
+            k,
+        )
+        if k == "validation":
+            prefix = "val"
+        else:
+            prefix = f"val-{k}"
```

Comment:
i'm not too keen on the magical strings that need to match everywhere. can you simply do:
```suggestion
        prefix = f"validation/{k}"
```